index.html

```diff
@@ -651,22 +650,22 @@ <h4>Try our tutorials and read the FAQ</h4>
                         </a>
                     </div>
                     <div class="tile-description">
-                        <h4>NewPipe is free and open source</h4>
+                        <h4>NewPipe is copylefted libre software</h4>
 
                         <p><strong>Your advantages</strong></p>
                         <ul class="check">
                             <li>Quick help when you have a problem (nobody's ever had one)</li>
-                            <li>Dozens of people bring NewPipe forward and translate it into <a href="https://hosted.weblate.org/engage/NewPipe/"><span data-newpipe-api="stats.translations">60+</span>&nbsp;languages</a>.</li>
+                            <li>Dozens of people bring NewPipe forward and translate it into <a href="https://hosted.weblate.org/engage/NewPipe/"><span data-newpipe-api="stats.translations">70+</span>&nbsp;languages</a>.</li>
                             <li>You can check the source code on <a href="https://github.com/TeamNewPipe/NewPipe">GitHub</a> and contribute to NewPipe.</li>
-                            <li>NewPipe is covered by the <a href="https://github.com/TeamNewPipe/NewPipe/blob/master/LICENSE">GPL&nbsp;3.0&nbsp;license</a>.</li>
-                            <li>Get NewPipe from <a href="https://f-droid.org/packages/org.schabi.newpipe/">F-Droid</a> and discover more FOSS apps.</li>
-                            <li class="text-success">NewPipe is available for free</li>
+                            <li>Licensed <a href="https://github.com/TeamNewPipe/NewPipe/blob/master/LICENSE">GPLv3+</a>.</li>
+                            <li>Get NewPipe from <a href="https://f-droid.org/packages/org.schabi.newpipe/">F-Droid</a> and discover more libre apps.</li>
+                            <li class="text-success">NewPipe is gratis, and welcomes donations by improving</li>
```

Comment:
donations in form of an improvement/PR, or donations in order to be able to improve it, which would be kinda wrong?

Reply:
Doesn't have to be a transitive verb